lib/adb.ts

```diff
@@ -73,24 +108,22 @@ class ADB {
 
     return new ADB(cloneOptions);
   }
-}
 
-ADB.createADB = async function createADB (opts) {
-  const adb = new ADB(opts);
-  adb.sdkRoot = await requireSdkRoot(adb.sdkRoot);
-  await adb.getAdbWithCorrectAdbPath();
-  try {
-    await adb.adbExec(['start-server']);
-  } catch (e) {
-    log.warn(e.stderr || e.message);
+  static async createADB(opts: Partial<ADBOptions>) {
+    const adb = new ADB(opts);
+    adb.sdkRoot = requireSdkRoot(adb.sdkRoot);
+    adb.getAdbWithCorrectAdbPath();
+    try {
+      await adb.adbExec(['start-server']);
+    } catch (e) {
+      const err = e as import('teen_process').ExecError;
+      log.warn(err.stderr || err.message);
+    }
+    return adb;
   }
-  return adb;
-};
+}
 
 // add all the methods to the ADB prototype
-for (const [fnName, fn] of _.toPairs(methods)) {
-  ADB.prototype[fnName] = fn;
-}
+Object.assign(ADB.prototype, methods);
```

Comment:
isn't this always complaining in eslint, that we cannot assign directly to the prototype?

Reply:
it is.  I think I should probably disable that warning, because I think we probably _should_ do it this way if we're using mixins.

Reply:
would you like to update that check in the global eslint config?